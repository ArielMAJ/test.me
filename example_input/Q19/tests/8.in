5
3 3
0.1 0.2 0.3
0.4 0.5 0.6
0.7 0.8 0.9
3 3
1.1 2.2 3.3
4.4 5.5 6.6
7.7 8.8 9.9
3 3
1.1 2.2 3.3
4.4 5.5 6.6
7.7 8.8 9.9
3 100
1.1 2.2 3.3 4.4 5.5 6.6 7.7 8.8 -9.9 0.1 1.1 2.2 3.3 4.4 5.5 6.6 7.7 8.8 9.9 0.1 1.1 2.2 3.3 4.4 5.5 6.6 7.7 8.8 9.9 0.1 1.1 2.2 3.3 4.4 5.5 6.6 7.7 8.8 9.9 0.1 1.1 2.2 3.3 4.4 5.5 6.6 7.7 8.8 9.9 0.1 1.1 2.2 3.3 -4.4 5.5 6.6 7.7 8.8 9.9 0.1 1.1 2.2 3.3 4.4 5.5 6.6 7.7 8.8 9.9 0.1 1.1 2.2 3.3 4.4 5.5 6.6 7.7 8.8 9.9 0.1 1.1 2.2 3.3 4.4 5.5 6.6 7.7 8.8 9.9 0.1 1.1 2.2 3.3 4.4 5.5 6.6 7.7 8.8 9.9 0.1
1.1 2.2 3.3 4.4 5.5 6.6 7.7 8.8 -9.9 0.1 6.6 7.7 8.8 -9.9 0.1 1.1 2.2 3.3 4.4 5.5 6.6 7.7 8.8 9.9 0.1 1.1 2.2 3.3 4.4 5.5 6.6 7.7 8.8 9.9 0.1 1.1 2.2 3.3 4.4 5.5 6.6 7.7 8.8 9.9 0.1 1.1 2.2 3.3 4.4 5.5 6.6 7.7 8.8 9.9 0.1 1.1 2.2 3.3 4.4 5.5 6.6 7.7 8.8 9.9 0.1 1.1 2.2 3.3 4.4 5.5 6.6 7.7 8.8 9.9 0.1 1.1 2.2 3.3 4.4 5.5 6.6 7.7 8.8 9.9 0.1 1.1 2.2 3.3 4.4 5.5 6.6 7.7 8.8 9.9 0.1 1.1 2.2 3.3 4.4 -5.5
1.1 2.2 3.3 4.4 5.5 6.6 7.7 8.8 9.9 0.1 1.1 2.2 3.3 4.4 5.5 6.6 7.7 8.8 9.9 0.1 1.1 2.2 3.3 4.4 5.5 6.6 7.7 8.8 9.9 0.1 1.1 2.2 3.3 4.4 5.5 6.6 7.7 8.8 9.9 0.1 1.1 2.2 3.3 4.4 5.5 6.6 7.7 8.8 9.9 0.1 1.1 2.2 3.3 4.4 5.5 -6.6 7.7 8.8 9.9 0.1 1.1 2.2 3.3 4.4 5.5 6.6 7.7 8.8 9.9 0.1 1.1 2.2 3.3 4.4 5.5 6.6 7.7 8.8 9.9 0.1 1.1 2.2 3.3 4.4 5.5 6.6 7.7 8.8 9.9 0.1 1.1 2.2 3.3 4.4 5.5 6.6 7.7 8.8 9.9 0.1
100 1
1.1
2.2
-3.3
4.4
5.5
6.6
7.7
8.8
9.9
0.1
6.6
7.7
8.8
9.9
0.1
1.1
2.2
3.3
4.4
5.5
6.6
7.7
8.8
9.9
0.1
1.1
2.2
3.3
4.4
5.5
6.6
7.7
8.8
-9.9
0.1
1.1
2.2
3.3
4.4
5.5
6.6
7.7
8.8
9.9
0.1
1.1
2.2
3.3
4.4
5.5
6.6
7.7
8.8
9.9
0.1
1.1
2.2
3.3
4.4
5.5
6.6
7.7
8.8
9.9
0.1
1.1
2.2
-3.3
4.4
5.5
6.6
7.7
8.8
9.9
0.1
1.1
2.2
3.3
4.4
5.5
6.6
7.7
8.8
9.9
0.1
1.1
2.2
3.3
4.4
5.5
6.6
7.7
8.8
9.9
0.1
1.1
2.2
3.3
4.4
-5.5