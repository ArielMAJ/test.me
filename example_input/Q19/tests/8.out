447995.2
1014934.2
1581873.3
